sympy/functions/elementary/complexes.py

```diff
@@ -896,18 +1064,32 @@ class periodic_argument(Function):
     logarithm. That is, given a period P, always return a value in
     (-P/2, P/2], by using exp(P*I) == 1.
 
+    Examples
+    ========
+
     >>> from sympy import exp, exp_polar, periodic_argument, unbranched_argument
     >>> from sympy import I, pi
     >>> unbranched_argument(exp(5*I*pi))
     pi
     >>> unbranched_argument(exp_polar(5*I*pi))
     5*pi
```

Comment:
All these examples may move to `unbranched_argument` where the documentation is missing.
However, you may have questions how to derive the summary, title there because the doc is completely missing there.

Reply:
I have updated the doc string. Is it fine?

Reply:
I think that it looks fine in unbranched_argument

Reply:
I'll remove those examples from periodic argument then.

Reply:
Done.